libparsec/crates/crypto/src/common/sequester.rs

```diff
@@ -43,6 +45,8 @@ pub(crate) fn serialize_with_armor(
     let (output_part, data_part) = others.split_at_mut(output.len());
 
     algorithm_part.copy_from_slice(algo.as_bytes());
+    at[0] = b'@';
+    size.copy_from_slice(&(key_size_bytes as u16).to_le_bytes());
```

Comment:
The standard for network stuff is to use Big endian for numeric value

https://stackoverflow.com/questions/13514614/why-is-network-byte-order-defined-to-be-big-endian
https://www.rfc-editor.org/rfc/rfc1700

Reply:
See: https://godbolt.org/z/jssb3n7xn
You are less efficient with big endian representation.
My professor told me that big endian was an error and it is used because it is easier for human but harder for computer.

Reply:
@touilleMan WDYT ?